tango_simlib/tests/test_sim_xmi_parser.py

```diff
@@ -238,6 +255,8 @@ def test_attribute_properties(self):
         attribute_data = self.xmi_parser.get_reformatted_device_attr_metadata()
 
         for attr_name, attr_metadata in attribute_data.items():
+            if attr_name == 'image1':
+                continue
```

Comment:
This is OK for now, but if we have more attributes in future that we know aren't added to the device, then we can make another list for them, similar to `EXPECTED_QUANTITIES_LIST`